modules/zeromq-ext/README.txt

```diff
@@ -0,0 +1,37 @@
+Apache Ignite ZeroMQ Streamer Module
+------------------------
+
+Apache Ignite ZeroMQ Streamer module provides streaming from ZeroMQ to Ignite cache.
+
+Starting data transfer to Ignite cache can be done with the following steps.
+
+1. Import Ignite ZeroMQ Streamer Module In Maven Project
+
+If you are using Maven to manage dependencies of your project, you can add ZeroMQ module
+dependency like this (replace '${ignite.version}' with actual Ignite version you are
+interested in):
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                        http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    ...
+    <dependencies>
+        ...
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-zeromq</artifactId>
+            <version>${ignite.version}</version>
+        </dependency>
+        ...
+    </dependencies>
```

Comment:
it's not true anymore and needs to be fixed.
but I like idea to match extension version and ignite version, so let's replace 1.0.0-SNAPSHOT with ignite version 2.9.0-SNAPSHOT.

Reply:
I have updated the docs